landing_page/.rubocop.yml

```diff
@@ -0,0 +1,139 @@
+AllCops:
+  Include:
+    - '**/*.rb'
+    - 'archive_posts.rake'
+  Exclude:
+    - bin/**/*
+    - config/developer_configs.rb
+    - config/developer_vars.rb
+    - db/**/*
+    - node_modules/**/*
+    - vendor/**/*
+  TargetRubyVersion: 2.5
+
+# Rails:
+#   Enabled: true
+
+# Rubocop Style Choices
+
+Layout/AccessModifierIndentation:
+  EnforcedStyle: outdent
+
+Layout/HashAlignment:
+  EnforcedLastArgumentHashStyle: ignore_implicit
+
+Layout/ParameterAlignment:
+  EnforcedStyle: with_fixed_indentation
+
+Layout/CaseIndentation:
+  EnforcedStyle: end
+  IndentOneStep: true
+
+Layout/DotPosition:
+  EnforcedStyle: trailing
+
+Layout/FirstArrayElementIndentation:
+  EnforcedStyle: consistent
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+Layout/MultilineOperationIndentation:
+  EnforcedStyle: indented
+
+Layout/EndAlignment:
+  EnforcedStyleAlignWith: variable
+
+Layout/LineLength:
+  Max: 240
+
+Metrics/ParameterLists:
+  CountKeywordArgs: false
+
+Metrics/BlockLength:
+  ExcludedMethods: ["context", "describe", "shared_examples"]
+
+Naming/PredicateName:
+    ForbiddenPrefixes: is_, have_
+
+Style/Alias:
+  EnforcedStyle: prefer_alias_method
+
+Style/AndOr:
+  EnforcedStyle: conditionals
+
+Style/ClassAndModuleChildren:
+  EnforcedStyle: compact
+  Exclude:
+    - app/controllers/api/**/*
+    - config/application.rb
+    - app/channels/application_cable/channel.rb
+    - app/channels/application_cable/connection.rb
+
+Style/Documentation:
+    Enabled: false
+
+Style/DoubleNegation:
+  Enabled: false
+
+Style/EmptyElse:
+  EnforcedStyle: empty
+
+Style/EmptyMethod:
+    Enabled: false
+
+Style/FrozenStringLiteralComment:
+    Enabled: false
+
+Style/HashEachMethods:
+    Enabled: false
+
+
+Style/HashSyntax:
+  EnforcedStyle: ruby19
+
+Style/HashTransformKeys:
+    Enabled: false
+
+Style/HashTransformValues:
+    Enabled: false
+
+Style/MultilineTernaryOperator:
+    Enabled: false
+
+Style/PercentLiteralDelimiters:
+  PreferredDelimiters:
+    "%i": ()
+    "%w": ()
+
+Style/RaiseArgs:
+  EnforcedStyle: compact
+
+Style/SingleLineMethods:
+  Enabled: false
+
+Style/SymbolArray:
+    Enabled: false
+
+# Temporary Rubocop TODO exclusions:
+# We might want to fix these at some point, but in the mean time we don't want to disable the cop for other files.
```

Comment:
Is this relevant to your code base? Have you assessed the settings below?

Reply:
I've now amended this, thanks for the feedback